### DISCUSSION TITLE
FPS changes from 120fps to 60fps when chnaging maps

### DISCUSSION THREAD

**Discussion by HugoLiuXB** [34736384/genshin-fps-unlock#421] [General]
120fps works fine when landing in any maps, if you can change to a diffrent maps like Pot, it changes to 60fps, only noticed this week so not sure if it's casued by recent patch

如果从提瓦特传送到尘歌壶, 120帧率失效, 似乎从任何一个地图转到另外一个地图都会发生这种情况

**Comment by ZUNSoft** [comment]
same situation

**Comment by majunb** [comment]
same, I notice it when I try to get artifacts.
